group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/GeneralUniformAssignmentBuilderTest.java

```diff
@@ -79,8 +80,16 @@ public void testTwoMembersNoTopicSubscription() {
             Collections.emptyMap()
         ));
 
-        AssignmentSpec assignmentSpec = new AssignmentSpec(members, HETEROGENEOUS);
-        GroupAssignment groupAssignment = assignor.assign(assignmentSpec, subscribedTopicMetadata);
+        GroupSpecImpl groupSpec = new GroupSpecImpl(
```

Comment:
nit: should all the `GroupSpecImpl groupSpec` initializations in the tests be `GroupSpec groupSpec`?